onnx2kerastl/operation_layers.py

```diff
@@ -585,3 +639,24 @@ def convert_trilu(node, params, layers, lambda_func, node_name, keras_name):
     else:
         result = tf.experimental.numpy.triu(input, k)
     layers[node_name] = result
+
+
+def convert_cumsum(node, params, layers, lambda_func, node_name, keras_name):
+    exclusive = bool(params.get("exclusive", 0))
+    reverse = bool(params.get("reverse", 0))
+    layers[node_name] = tf.math.cumsum(layers[node.input[0]], layers[node.input[1]],
+                                       exclusive=exclusive, reverse=reverse)
+
+
+def convert_is_inf(node, params, layers, lambda_func, node_name, keras_name):
+    if params.get("detect_negative") is not None or params.get("detect_negative") is not None:
+        raise AttributeError("Unsupported params detected in isInf conversion: detect_negative/detect_positive")
```

Comment:
I think we can detect if the inf is negative or positive (maybe just checking if bigger than 0) so we don't have to throw an exception here

Reply:
I don't think these params can ever propogate from pytorch. I suggest that if we'll encounter them we'll solve it then.